modules/op-setting-annotations-labels-namespace.adoc

```diff
@@ -0,0 +1,28 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-setting-annotations-labels-namespace_{context}"]
+= Setting labels and annotations for the {pipelines-shortname} installation namespace
+
+You can set labels and annotations on the `openshift-pipelines` namespace in which the operator installs {pipelines-shortname}.
+
+[NOTE]
+====
+Changing the name of the `openshift-pipelines` namespace is not supported at present
+====
+
+To set the labels and annotations, use the `spec.targetNamespaceMetadata` specification in the `TektonConfig` custom resource (CR).
+
+.Example of setting labels and annotations for the `openshift-pipelines` namespace
```

Comment:
```suggestion
.Example `TektonConfig` CR
```
IMO, this can be kept short because heading and description already indicate users about setting labels and annotations.
Up to you though!

Reply:
There are many examples of this same CR in this module, so I do want to keep this more specific

Reply:
Okay sounds good.